mdf_matio/adapters/basic_adapters.py

```diff
@@ -0,0 +1,60 @@
+from mdf_matio.adapters.generic import GenericMDFAdapter
+
+
+# Adapters that require no extra processing and can just use the GenericMDFAdapter
+
+class CrystalStructureAdapter(GenericMDFAdapter):
+    """Adapt the CrystalStructureExtractor"""
+    # No extra processing required
+    pass
+
+
+class ElectronMicroscopyAdapter(GenericMDFAdapter):
+    """Adapt the ElectronMicroscopyExtractor"""
+    # No extra processing required
+    pass
+
+
+class GenericFileAdapter(GenericMDFAdapter):
+    """Adapt the GenericFileExtractor"""
+    # No extra processing required
+    pass
+
+
+class FilenameAdapter(GenericMDFAdapter):
```

Comment:
What is the filename parser? Is it different than the "generic parser"

Reply:
Yes, the `FilenameParser` extracts metadata from a filename (not the contents of a file). I can't remember if this made it into MatIO yet, or if there were issues, but a filename extractor exists in MDF right now, separate from the file info extractor (which extracts file size, type, path, etc.).

Reply:
Cool!